README.md

```diff
@@ -1,703 +1,48 @@
 
-<p align="center">
-  Create production-ready GraphQL servers
-</p>
-<p align="center">
-  <a href="https://travis-ci.org/lucasbento/create-graphql"><img src="https://travis-ci.org/lucasbento/create-graphql.svg?branch=master"></a>
-  <a href="https://ci.appveyor.com/project/lucasbento/create-graphql/branch/master"><img src="https://ci.appveyor.com/api/projects/status/cpxul2ofnyf6ypy8/branch/master?svg=true"></a>
-  <a href="https://codecov.io/gh/lucasbento/create-graphql"><img src="https://img.shields.io/codecov/c/github/lucasbento/create-graphql.svg"></a>
-  <a href="https://github.com/airbnb/javascript"><img src="https://img.shields.io/badge/code%20style-airbnb-blue.svg"></a>
-  <a href="https://github.com/lucasbento/create-graphql/issues"><img src="https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat"></a>
-</p>
 
 ## About
-[Check out the post on Medium!](https://medium.com/entria/announcing-create-graphql-17bdd81b9f96#.idrwtqa9k) 😉
+**[Create-GraphQL](https://github.com/lucasbento/create-graphql)** is a command-line utility to build production-ready servers with GraphQL and also generate *Mutations*, *Types* and more into existent projects
+
+Check out the post *[Announcing Create-GraphQL](https://medium.com/entria/announcing-create-graphql-17bdd81b9f96)* on **[Entria](https://medium.com/entria)** medium
 
 ## Install
 
+#### With Yarn:
 ```sh
-npm install --global create-graphql
+yarn global add create-graphql
 ```
 
-## Update
+#### With NPM:
 ```sh
-npm update --global create-graphql
+npm install --global create-graphql
 ```
 
 ## Usage
-
-It can generate:
-
-- [x] Mutations (create and edit);
-- [x] Types;
-- [x] Connections;
-- [x] Loaders;
-- [x] Tests (with [Jest](https://github.com/facebook/jest));
-- [x] Generate from a [Mongoose](https://github.com/Automattic/mongoose) schema (with `--schema` option).
-
-### Examples
-
-- Generate a type, the file name will be automatically suffixed with `Type`:
-  ```sh
-  create-graphql generate --type Awesome # create `AwesomeType`.
-  ```
-
-- Generate mutations, the files names will be automatically suffixed with `Mutation`:
-  ```sh
-  create-graphql generate --mutation Awesome # create `AwesomeAddMutation` & `AwesomeEditMutation`.
-  ```
-
-- Generate mutations based on a Mongoose schema:
-  ```sh
-  create-graphql generate --mutation Awesome --schema Awesome
-  ```
-  Which will look for `Awesome` Mongoose schema file in `./src/model` and generate the mutations fields based on it.
-
-🔥 **Tip**: You may use aliases commands to create multiple files in one single command:
-
+You can create a brand new GraphQL project:
```

Comment:
How do you plan it to look like?

Create-GraphQL's takes a little more space than that gif.